en/lessons/basics/enum.md

```diff
@@ -184,11 +184,13 @@ iex> Enum.sort([%{:count => 4}, %{:count => 1}])
 [%{count: 1}, %{count: 4}]
 ```
 
-### uniq
+### uniq_by
 
-We can use `uniq/1` to remove duplicates from our enumerables:
+We can use `uniq_by/2` to remove duplicates from our enumerables:
 
 ```elixir
-iex> Enum.uniq([1, 2, 3, 2, 1, 1, 1, 1, 1])
+iex> Enum.uniq_by([1, 2, 3, 2, 1, 1, 1, 1, 1], fn x -> x end)
 [1, 2, 3]
 ```
+
+This function was previously known as `uniq/1`, which is obsolete from Elixir 1.4, but is still available (with alerts).
```

Comment:
Let's just make the change from `uniq/1` to `uniq_by/2` and skip this note, that way we don't need to come back later and remove it 👍